doc/source/admin/scaling.md

```diff
@@ -376,16 +376,8 @@ gravity:
 ```
 
 In this example 4 processes will be started in total:
-3 processes will act as job handlers and workflow schedulers, and one process will be dedicated to handling jobs for the `special` tag only. With the `job_conf.xml` configuration above these would be jobs created by the `test1` tool.
-You can omit the `pools` argument, this will then default to:
-
-```yaml
```

Comment:
Did you mean to remove this, and if yes, why ?

Reply:
Only line 380, about the `pools` argument, as it does not work in practice. I guess rebasing killed line 379 also.

Reply:
No it is actually reformatted below. So yes, I intended to remove this, as it is misleading and caused some confusion for me at least, and probably others

Reply:
well, it is how it's supposed to work, can you leave that in please ?

Reply:
It doesn't work. I just tested now commenting out the pools part from my galaxy.yml and all jobs are now grey.

I think the most important part, though, is that the user has some example config to look at for setting up dynamic job handlers that actually work. I have spent significant time at multiple occasions trying to figure out what is wrong due to this issue (and then obviously forgetting the details until next time). But as long as there are some example config available that actually work, I am fine with leaving this part of the documentation in there and I can submit a separate issue on this.

We could even add the link to the issue in the docs with a note that it is supposed to work, but not right now! 🙂

Reply:
> all jobs are now grey.

there's a million other things that could have gone wrong, and if so they need fixing, not removing documention. Can you please focus just on the changes for this PR.

Reply:
> there's a million other things that could have gone wrong, and if so they need fixing, not removing documentation. Can you please focus just on the changes for this PR.

So this PR is in essence about making it easier for people to get interactive tools to run so that they do not need to spend as much time as I have had to do to make things work, and that includes improving documentation and config examples. So I would say it is on topic. But removing this part of the documentation was clearly the wrong move, so I will revert that.

Reply:
I get that, but it does work in the sense that you don't have to specify this, otherwise `run.sh` would fail to run jobs or workflows. If you could open an issue describing what part of this doesn't work that would be great.